File: apps/chat-android/messaging-websocket/src/main/java/com/amazonaws/services/chime/sdk/messaging/session/MessagingSessionConfiguration.kt
/*
 * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 * SPDX-License-Identifier: MIT-0
 */

package com.amazonaws.services.chime.sdk.messaging.session

import kotlin.random.Random
import kotlin.random.nextUInt

data class MessagingSessionConfiguration(
    val userArn: String,
    val endpointUrl: String,
    val region: String,
    val credentials: ChimeUserCredentials,
    val messagingSessionId: String = Random.nextUInt().toString()
)

File: apps/chat-ios/AmazonChimeSDKMessagingDemo/messaging-websocket/session/MessagingSessionConfiguration.swift
//
//  MessagingSessionConfiguration.swift
//  AmazonChimeSDKMessagingDemo
//
//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
//  SPDX-License-Identifier: MIT-0
//

import Foundation

struct MessagingSessionConfiguration {
    var userArn: String
    var endpointUrl: String
    var region: String
    var credentials: ChimeUserCredentials
    var messagingSessionId: String = UUID().uuidString
}
